app/View/Dictionaries/buttons.inc

```diff
@@ -37,7 +37,10 @@ if(!$e && !$permissions['view'])
 
 if($e) {
   // Manage Dictionary Entries button
-  if($this->action == "edit" && $permissions['edit']){
+  // We only allow Dictionary Entries to be managed for Standard Dictionaries.
+  if($this->action == "edit"
+     && $permissions['edit']
+     && $dictionaries[0]['Dictionary']['id'] === DictionaryModeEnum::Standard){
```

Comment:
Sorry I just noticed this, I think you want to check `mode` here?

```suggestion
     && $dictionaries[0]['Dictionary']['mode'] === DictionaryModeEnum::Standard){
```

Reply:
Done